package.json

```diff
@@ -1807,22 +1807,26 @@
                 "docker.host": {
                     "type": "string",
                     "default": "",
-                    "description": "%vscode-docker.config.docker.host%"
+                    "description": "%vscode-docker.config.docker.host%",
+                    "scope": "machine-overridable"
```

Comment:
Can this be applied to other settings (ex: docker.volumes.sortBy). Looks like most of them eligible for machine level settings.

Reply:
Those don't seem to me that they should be machine-scoped. This scoping is specifically to keep remote and local settings apart, i.e. if you have `docker.host` set locally, today it will be picked up in the remote environment; with this setting it will no longer be picked up in the remote environment.